index.bs

```diff
@@ -492,22 +511,34 @@ To <dfn>trigger attribution</dfn> given an [=attribution trigger=] |trigger| run
 1. Set |matchingSources| to the result of [=list/sort in descending order|sorting=] |matchingSources|
     in descending order, with |a| being less than |b| if any of the following are true:
       * |a|'s [=attribution source/priority=] is less than |b|'s [=attribution source/priority=].
-      * |a|'s [=attribution source/priority=] is equal to |b|'s [=attribution source/priority=] and |a|'s 
+      * |a|'s [=attribution source/priority=] is equal to |b|'s [=attribution source/priority=] and |a|'s
          [=attribution source/source time=] is less than |b|'s [=attribution source/source time=].
 1. Let |sourceToAttribute| be the first item in |matchingSources|.
 1. If |trigger|'s [=attribution trigger/dedup key=] is not null and |sourceToAttribute|'s
     [=attribution source/dedup keys=] [=list/contains=] it, return.
+1. Let |report| be the result of running [=obtain a report=] with |sourceToAttribute| and |trigger|.
+1. If |sourceToAttribute|'s [=attribution source/number of reports=] value is equal to the
+    user agent's [=max reports per source=] value, then:
+    1. Let |matchingReports| be all entries in the [=attribution report cache=] where all of the following are true:
```

Comment:
Do you think it's worthwhile to split this into a different algorithm? 

I'm thinking, if reports is equal to max reports, let removed be the the result of running "maybe remove lowest priority report"...

Reply:
Seems unnecessary for the time being, though if there's any additional complexity we can do so.